### ISSUE TITLE
Community connector page creation: make the nested options a separate question

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-in-action/platform#947]
Nested options in some of the questions, such as age groups, gender groups, ethnoracial groups, gender groups, etc. are not accessible for different ATs. We had Tab navigation and screen reader navigation issues with these nested options. All users including magnifier users and Dragon users were not sure if they are required to pick these nested options or not. It would be more accessible to remove these nested options and ask them as a separate question. 

We take a similar approach when we ask "Can you connect to people with disabilities, Deaf persons, and/or their supporters? (required)" If they select "People who experience disabilities" we show a subsequent question " Please select people with disabilities that you can connect to (required)" instead of nesting it inside the first question.

**Comment by cherylhjli** [comment]
Individual - community connector:
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2613%3A68551

Community organization - rep org:
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2560%3A65260

Community org - support org:
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2602%3A68633

Community org - civil society:
https://www.figma.com/file/AXufBnFAVHvGdsxnQMVhNF/The-Accessibility-Exchange-Wireframes?node-id=2567%3A65306

**Comment by greatislander** [comment]
See also: https://github.com/accessibility-exchange/platform/pull/1288#issuecomment-1310796711
